lona/html/node_list.py

```diff
@@ -181,16 +185,21 @@ def __contains__(self, node):
             return node in self._nodes
 
     # serialization ###########################################################
-    def _reset(self, value):
-        if not isinstance(value, list):
-            value = [value]
+    def _reset(self, values):
+        if not isinstance(values, list):
+            values = [values]
 
         with self._node.lock:
+            for node in self._nodes:
+                node._set_parent(None)
```

Comment:
Looks like need to do the same in `__setitem__`

Reply:
Oh! You are right! Thanks for catching that. I added a patch that adds a similar change to `__setitem__`.